### ISSUE TITLE
Increasing browser memory usage for long running web sites

### ISSUE THREAD

**Issue by fubar-coder** [praeclarum/Ooui#25]
After seeing that all created nodes are kept in a global variable, I fear that the browser memory usage might be uncontrollable when the application gets more complex - especially when developing a SPA.

**Comment by praeclarum** [comment]
Yes I agree - that global store can be improved. Ideally it could be removed!

**Comment by touseefbsb** [comment]
variables should be ideally databound and limited only to the scope of a page or a specific function, just like in traditional xaml/c# approach.

**Comment by praeclarum** [comment]
Yes I did plan on removing the globals in preference for a closure so that Ooui can be embedded in Razor templates. That won't solve the global store, but is a start.

What I really need are *weak references* in JavaScript.
